src/network/invthread.py

```diff
@@ -78,7 +78,7 @@ def run(self):  # pylint: disable=too-many-branches
                             if connection == dandelion.instance.objectChildStem(inv[1]):
                                 # Fluff trigger by RNG
                                 # auto-ignore if config set to 0, i.e. dandelion is off
-                                if random.randint(1, 100) >= state.dandelion_enabled:  # nosec B311
+                                if random.randint(1, 100) >= dandelion.instance.enabled:  # nosec B311
```

Comment:
Hmm, I forgot, the value is actually an int between 0 and 100. We need to rethink.

Reply:
when dandelion is on what would be the probable return value. that help us resolve the fixing comparison between `int` and `bool`